test/app.ts

```diff
@@ -1,13 +1,12 @@
-
 import express = require('express');
 import router from './router';
 
 const app: express.Application = express();
 
 // requesthandler
-const requestHandler: express.RequestHandler = (req, res, next) => {
+const requestHandler: express.RequestHandler = (req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Comment:
Yeah but on master the type is not infered to `any` but properly to `express.Request` etc. Did your changes change that? If yes that would be a regression and breaking change.